en/case-studies/rwanda-rural-water.rst

```diff
@@ -0,0 +1,111 @@
+
+Goal
+----
+
+To conduct data collection of all rural water supply network in Rwanda, 
+and keep updating the data continuously in order to improve operation & 
+maintenance of waterworks.
+
+Project preparation
+-------------------
+
+Before starting our data collection, we conducted the following things.
+ - Develop our own PostGIS database
+ - Develop QGIS project template with Geopackage. The Geopackage table design should be equal to PostGIS. So we can just copy and paste to PostGIS.
+
+Apart from preparing Android devices, if possible, you may purchase GPS devices. In WASAC, we bought Garmin GPSMAP 64S. 
+Sometimes, GPS of smartphone and tablet is not very accurate, so we normally capture the same location by using Garmin GPS, then correct the location of QField's data after data collection work.
+
+Data collection
+---------------
+
+Once we prepared Geopackage and QGIS project template, 
+we conducted training of QGIS/QField in July 2018 and 
+launched our data collection work in 27 districts in the whole country of Rwanda. 
+27 engineers sent their Geopackage to central office in Kigali, 
+the MIS specialist validated and entered their data from Geopackage to PostGIS database. 
+We completed our initial data collection works until April 2019.
+
+.. container:: clearer text-center
+
+   .. image:: /images/rwanda-rural-water-1.png
+      :width: 600px
+      :alt: data collection procedure
+
+Data distribution and updating
+------------------------------
+
+The most significant thing after data collection is ``updating``. 
+We have seen many organization in Africa, they normally failed to keep updating. 
+Several years later, their data will normally become too old, 
+and most of them need to put efforts on data collection again. 
+
+However, WASAC is continuously updating all of the data until now. 
+We think QField works very well for this purpose.
+
+In order to distribute and updating the data, 
+we developed a python script `postgis2qfield <https://github.com/WASAC/postgis2qfield>`__. 
+This ``postgis2qfield`` tool can extract the data from PostGIS and create Geopackages for each districts in Rwanda. 
+We upload these 27 geopackage together with QGIS project template to Google Drive storage. 
+After that, those engineers in districts download their geopakage to Android device to continue adding and updating the data. 
+Once they completed updating, send geopackage to central office again, MIS specialist update PostGIS database and regenerate geopackages for QField.
+
+.. container:: clearer text-center
+
+   .. image:: /images/rwanda-rural-water-2.png
+      :width: 600px
+      :alt: data distribution and updating procedure
+
+Why didn't we use ``QFieldSync`` plugin for this purpose? 
+
+Although QField has a nice QFieldSync plugin for linking between PostGIS and Geopackage, 
+we didn't select QFieldSync. Because QFieldSync plugin only can create a Geopackage only, 
```

Comment:
That's actually not completely true, QFieldSync can prepare a subset of the data only. But it's always a bounding box / rectangular.

Reply:
I wonder if we could just remove this paragraph? It highlights drawbacks of the QFieldSync plugin. One the one hand it's not completely correct (see above) and they will likely be inappropriate in the near future and start to be misleading for people reading this.

Reply:
> But it's always a bounding box / rectangular

Sure, I remember QFieldSync has this function. However, in WASAC, they need to separate data exactly by the boundary of district. Also, it is not easy for them to repeat the same process for all the districts. That is why I developed customized python script.

> I wonder if we could just remove this paragraph? 

Yeah, I think I can remove this paragraph.

Reply:
I deleted the paragraph in this commit(https://github.com/opengisch/QField-docs/pull/77/commits/088ffee04cd7ee0a23b4d85e222b66ddb00452eb).